### PR TITLE
Add support for multiple CA certificates

### PR DESCRIPTION
**Pull request by mwang3-wrs**
This change allows multiple CA certificates to be installed by
specifying them in the deployment config.

Signed-off-by: Melissa Wang <melissa.wang@windriver.com>

### CODE REVIEW

File: pkg/controller/system/system_controller.go [hunk 1]
```diff
@@ -848,8 +848,9 @@ func (r *ReconcileSystem) PrivateKeyTranmissionAllowed(client *gophercloud.Servi
 // ReconcileCertificates configures the system certificates to align with the
 // desired list of certificates.
 func (r *ReconcileSystem) ReconcileCertificates(client *gophercloud.ServiceClient, instance *starlingxv1.System, spec *starlingxv1.SystemSpec, info *v1info.SystemInfo) error {
+
```
Comment (donpenney) [inline pkg/controller/system/system_controller.go:851]: The blank line seems out of place. Was this added unintentionally?

Comment (mwang3-wrs) [reply pkg/controller/system/system_controller.go:851]: Yes, I'll remove that.

File: pkg/apis/starlingx/v1/system_types.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ type CertificateInfo struct {
 // point to a Secret named by the system.
 func (in *CertificateInfo) DeepEqual(other *CertificateInfo) bool {
 	if other != nil {
-		return in.Type == other.Type
+		return (in.Type == other.Type) && (in.Secret == other.Secret)
```
Comment (donpenney) [inline pkg/apis/starlingx/v1/system_types.go:60]: Per discussion, it appears the Secret value from the model does not align with the Secret value from sysinv, so this field would not be able to be used for comparing items

**Comment by mwang3-wrs** [comment]
Gophercloud changes: https://github.com/Wind-River/gophercloud/pull/7

**Pull request merged**